src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -7322,6 +7322,58 @@ public class MyComponent : ComponentBase
         CompileToAssembly(generated);
     }
 
+    [Fact] // https://github.com/dotnet/razor/issues/8170
```

Comment:
Hmm, if we don't already have `[WorkItem(...)]` attribute in this repo we should add it and use it here. Many times in the past we've gotten an advantage by having related issues be greppable in the system.

Reply:
I could do that in separate PR. I already added bunch of these comments in my bug fixes, so I would replace them all.